### DISCUSSION TITLE
User Rank Statistics > the top 500?

### DISCUSSION THREAD

**Discussion by Tom-H-L** [codewars/codewars.com#2959] [Q&A] [answered]
Is there some site where someone can see all user data statistics from Codewars, i.e. the amount of users per kyu rank, total users, (daily) active users, etc. that goes beyond only the top 500 users found here: [https://www.codewars.com/users/leaderboard/ranks](https://www.codewars.com/users/leaderboard/ranks) ?

I would love to see how many users are in my kyu group, understand what it will take me to reach the top 500 charts, etc.

**Comment by hobovsky** [comment, marked as answer]
Some information about specific  users can be found with this API endpoint: `https://www.codewars.com/api/v1/users/{user}`

Some information about your possibilities of ranking up can be found using Codewars Discord bot and its `/rankup` command.

![image](https://user-images.githubusercontent.com/23709795/224106660-9c82e98f-c7a0-49f6-a361-cee52a6b4367.png)

There is no place to find aggregated stats of users other than top 500 leaderboards.

**Reply by Tom-H-L** [reply]
Thank you for the information! Would be great to see info for > 500 and aggregated stats in general.
